### ISSUE TITLE
Level 2 of the official addon not completable

### ISSUE THREAD

**Issue by MiroslavR** [Panzerschrek/Chasm-Reverse#12]
Savegame: [save_02.pcs.zip](https://github.com/Panzerschrek/Chasm-Reverse/files/1531660/save_02.pcs.zip)

You can't pull the switch to continue.

**Comment by MiroslavR** [comment]
It seems that `link  23 33 1` and `return 23 33 1` cancel each other out and the process/procedure does not get a chance to run.

However, the process _does_ run in vanilla Chasm and that return appears to make reversing the switch state possible. With the return omitted, PanzerChasm and vanilla behave equally (but you can't reverse the switch).

Here's the process definition:
```
#proc 01
 startdelay 0
 lightremap
 linkswitchat 23 33
 speed 12
 sfxposxy 14 25
 sfxid 80
#action
 move 14 25 0 -480
 move 14 26 0 -480
 move 13 26 0 -480
 move 14 28 0 -480

 move 14 41 0 -480
 move 15 41 0 -480
 move 13 40 0 -480

 change 13 41 217
 change 14 27 167
#end
```

**Comment by Panzerschrek** [comment]
Yes, this script is very strange. I have no idea, how to fix this.
